.github/workflows/release.yml

```diff
@@ -58,13 +58,13 @@ jobs:
 
       - name: Update dependency versions for the samples
         run: |
-          ./mvnw org.codehaus.mojo:versions-maven-plugin:2.16.2:set -DnewVersion=${{steps.metadata.outputs.current-version}} -pl samples/email-a-poem
-          ./mvnw org.codehaus.mojo:versions-maven-plugin:2.16.2:set -DnewVersion=${{steps.metadata.outputs.current-version}} -pl samples/fraud-detection
-          ./mvnw org.codehaus.mojo:versions-maven-plugin:2.16.2:set -DnewVersion=${{steps.metadata.outputs.current-version}} -pl samples/review-triage
-          ./mvnw org.codehaus.mojo:versions-maven-plugin:2.16.2:set -DnewVersion=${{steps.metadata.outputs.current-version}} -pl samples/chatbot
-          ./mvnw org.codehaus.mojo:versions-maven-plugin:2.16.2:set -DnewVersion=${{steps.metadata.outputs.current-version}} -pl samples/csv-chatbot
-          ./mvnw org.codehaus.mojo:versions-maven-plugin:2.16.2:set -DnewVersion=${{steps.metadata.outputs.current-version}} -pl samples/cli-translator
-          ./mvnw org.codehaus.mojo:versions-maven-plugin:2.16.2:set -DnewVersion=${{steps.metadata.outputs.current-version}} -pl samples/chatbot-easy-rag
+          ./mvnw org.codehaus.mojo:versions-maven-plugin:2.16.2:set -DnewVersion=${{steps.metadata.outputs.current-version}} -f samples/email-a-poem
+          ./mvnw org.codehaus.mojo:versions-maven-plugin:2.16.2:set -DnewVersion=${{steps.metadata.outputs.current-version}} -f samples/fraud-detection
+          ./mvnw org.codehaus.mojo:versions-maven-plugin:2.16.2:set -DnewVersion=${{steps.metadata.outputs.current-version}} -f samples/review-triage
+          ./mvnw org.codehaus.mojo:versions-maven-plugin:2.16.2:set -DnewVersion=${{steps.metadata.outputs.current-version}} -f samples/chatbot
+          ./mvnw org.codehaus.mojo:versions-maven-plugin:2.16.2:set -DnewVersion=${{steps.metadata.outputs.current-version}} -f samples/csv-chatbot
+          ./mvnw org.codehaus.mojo:versions-maven-plugin:2.16.2:set -DnewVersion=${{steps.metadata.outputs.current-version}} -f samples/cli-translator
+          ./mvnw org.codehaus.mojo:versions-maven-plugin:2.16.2:set -DnewVersion=${{steps.metadata.outputs.current-version}} -f samples/chatbot-easy-rag
```

Comment:
Probably a good idea to use some bash to automate it?

```
for module in samples/*; do echo $module; done
```

Make sure you define the shell as bash.

Reply:
I thought about that too, but there's one sample (the jbang one) that isn't maven based so we'd have to exclude it, and that felt ugly enough for me to persuade me to go with hardcoding the list of modules. But of course we can do it.

Reply:
If you insist:

```
for pom in `find samples/ -name pom.xml`; do module=$(dirname "$pom"); echo $module; done
```

Reply:
@iocanel do you want to update the PR as per Guillaume's suggestion?

Reply:
I have added a comment below